ghastly/__about__.py

```diff
@@ -1,3 +1,3 @@
 
 
-__version__ = "0.1.0"
+__version__ = "0.3.0"
```

Comment:
No version 0.2.0?

Reply:
I could have sworn last version was 0.2.0, but it looks like I was wrong.  I can set it back to 0.2.0